icaruscode/CRT/CRTUtils/CRTPMTMatchingUtils.cxx

```diff
@@ -63,40 +67,67 @@ icarus::crt::CRTMatches icarus::crt::CRTHitmatched(
       .R();
     if (abs(tof) >= interval) continue;
     if (tof < 0) {
-      if (crtHit->plane > 36)
+      if (MatchBottomCRT && crtHit->plane > 49) 
```

Comment:
I suggest that you instead add a check at the beginning of the loop:
```cpp
    if (!MatchBottomCRT && crtHit->plane > 49) continue;
```
(I assume that `50` is the first bottom "plane" and `49` is not a bottom plane) so that we just skip those hits completely.

Reply:
exactly!! thank you